### PR TITLE
Ek

### PR DESCRIPTION
**Pull request by ekuiter**

### CODE REVIEW

File: MYshort.bib [hunk 1]
```diff
@@ -353,6 +353,7 @@ @String{PPL
 @String{PNAS = "PNAS"}
 @String{QREI = "QREI"}
 @String{REJ = "REJ"}
+@String{SA = "Sci. Adv."}
```
Comment (pmbittner) [inline MYshort.bib:356]: This abbreviation is inconsistent with our other abbreviations (which for quite some journals are actually not the official ones as documented for JSME below for example). So I guess sticking to the actual abbreviation here makes sense. So I think there is need for changes here but maybe we should open an Issue here? Maybe having a third MYxxxx.bib for real journal abbreviations could make sense? :thinking: I am undecided on how to proceed here but it should probably be made consistent somehown.

File: literature.bib [hunk 2]
```diff
@@ -326,6 +326,17 @@ @mastersthesis{Guething23
 
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%% 2024 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 
+@article{KCP+:SA24,
+	author  = {Sayash Kapoor and Emily M. Cantrell and Kenny Peng and Thanh Hien Pham and Christopher A. Bail and Odd Erik Gundersen and Jake M. Hofman and Jessica Hullman and Michael A. Lones and Momin M. Malik and Priyanka Nanayakkara and Russell A. Poldrack and Inioluwa Deborah Raji and Michael Roberts and Matthew J. Salganik and Marta Serra-Garcia and Brandon M. Stewart and Gilles Vandewiele and Arvind Narayanan},
+	title   = {{REFORMS: Consensus-based Recommendations for Machine-learning-based Science}},
```
Comment (pmbittner) [inline literature.bib:331]: capizalizemytitle.com says `REFORMS: Consensus-Based Recommendations for Machine-Learning-Based Science` to me

**Review by pmbittner — CHANGES_REQUESTED** [review]

File: literature.bib (hunk 2 above)
Comment (ekuiter) [reply literature.bib:331]: good catch

File: MYshort.bib (hunk 1 above)
Comment (ekuiter) [reply MYshort.bib]: I changed this to SciAdv, is this okay?

Comment (pmbittner) [reply MYshort.bib]: I think that is a good compromise for now. :) Thanks for opening the respective issue!

**Review by pmbittner — APPROVED** [review]

**Comment by pmbittner** [comment]
Fell free to merge once the merge conflicts for `.gitignore` are resolved. :)

**Pull request merged**